src/Asset/ManifestLookupInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\Asset;
+
+
+interface ManifestLookupInterface
+{
+    public function getManifestPath(string $path): ?string;
```

Comment:
Same as the EntrypointLookupInterface: let's make this return `string` (not `?string`) and create a new `ManifestKeyNotFoundException` that we should throw if the key does not exist (and rename `$path` to `$key`).